Dockerfile

```diff
@@ -1,14 +1,34 @@
-FROM docker.io/library/node:16.14.2 AS build
+FROM docker.io/library/node:16.14.2 AS web-builder
 
 WORKDIR /usr/src/app
 
-COPY package*.json .
-RUN npm ci && npm cache clean --force
+COPY web/package*.json web/
+COPY Makefile Makefile
+RUN make install-frontend-ci-clean
 
-COPY . /usr/src/app
-RUN npm run build
+COPY web/ /usr/src/app/web/
+RUN make build-frontend
 
-FROM docker.io/library/nginx:stable
+FROM docker.io/library/golang:1.18 as go-builder
 
-RUN chmod g+rwx /var/cache/nginx /var/run /var/log/nginx
-COPY --from=build /usr/src/app/dist /usr/share/nginx/html
+WORKDIR /opt/app-root
+
+COPY Makefile Makefile
+COPY go.mod go.mod
+COPY go.sum go.sum
+
+RUN make install-backend
+
+COPY config/ config/
+COPY cmd/ cmd/
+COPY pkg/ pkg/
+
+RUN make build-backend
+
+FROM docker.io/library/debian:11-slim
```

Comment:
any reason not to use RH ubi micro or similar https://catalog.redhat.com/software/containers/ubi8-micro/601a84aadd19c7786c47c8ea?container-tabs=overview